unpack_test.go

```diff
@@ -146,7 +146,9 @@ func TestReifyUnpackers(t *testing.T) {
 	}
 
 	// apply configurations
-	for _, c := range configs {
+	for i, c := range configs {
+		t.Logf("Test config (%v): %#v", i, c)
```

Comment:
Some code in go-ucfg is somewhat old (before `t.Run` was introduced to the stdlib). Better introduce `t.Run` when updating/modifying tests.

Reply:
I will fix the wording on the log message. This test is a little different because the asserts are not in the for loop, they are actually after it. The configs are actually applied to the same structure multiple times, then the values after the multiple calls to Unpack are checked.